utilities/transactions/write_prepared_txn.cc

```diff
@@ -63,6 +62,56 @@ Iterator* WritePreparedTxn::GetIterator(const ReadOptions& options,
   return write_batch_.NewIteratorWithBase(column_family, db_iter);
 }
 
+struct CFKeyComparator {
+  bool operator()(const std::pair<uint32_t, Slice>& lhs,
+                  const std::pair<uint32_t, Slice>& rhs) const {
+    if (lhs.first != rhs.first) {
+      return lhs.first < rhs.first;
+    }
+    return lhs.second.compare(rhs.second) < 0;
+  }
+};
+// Count the number of sub-batches inside a batch. A sub-batch does not have
+// duplicate keys.
+struct SubBatchCounter : public WriteBatch::Handler {
+  std::set<std::pair<uint32_t, Slice>, CFKeyComparator> keys_;
+  size_t batches_;
+  SubBatchCounter() : batches_(1) {}
+  size_t BatchCnt() { return batches_; }
```

Comment:
nit: s/BatchCnt/BatchCount/ ? No need to shorten names that's not long..

Reply:
sure.